profiles/serializers.py

```diff
@@ -204,6 +204,15 @@ class Meta:
         )
         read_only_fields = ("person",)
 
+    def validate_for_delete(self, data):
+        password = data.get("password")
+        if not password:
+            raise serializers.ValidationError("Password is required")
```

Comment:
just write `required=True` in the serializer field, and there will be no need to make it 'manually'.

Reply:
Fixed